README.md

```diff
@@ -32,13 +32,17 @@ Synopsis of Active Draco Packages
   common abstraction for objects and libraries that return material data
   (opacities, atomic cross sections, equation-of-state data, etc.)
 * cdi_analytic - Analytic models for physical data
+* cdi_CPEloss - Energy straggling models for charged particles.
```

Comment:
Can we change to "Stopping power models for charged particles."? I think "straggling" implies stochastic fluctuations in stopping power, which we aren't currently modeling.

Reply:
Updated in b777da7e4240af9cb16862de082bd9ce115bdc4d